### ISSUE TITLE
loss=nan and experiencing training oscillations and high loss values

### ISSUE THREAD

**Issue by JDBSK** [Ashish-Paka/Visual-Odometry-with-upgraded-DinoVO#2]
Hello, your work is very interesting and great. I have encountered the same problem as you did before, with a training loss of nan. But I don't know where the issue lies. Could you please explain how you locate the problem? Is it by outputting all the gradients to locate the position of the problem?
